packages/vite/src/node/plugins/importAnalysisBuild.ts

```diff
@@ -97,8 +97,9 @@ export function buildImportAnalysisPlugin(config: ResolvedConfig): Plugin {
 
     async transform(source, importer) {
       if (
-        importer.includes('node_modules') &&
-        !source.includes('import.meta.glob')
+        config.build.lib ||
```

Comment:
Think we need to check if the current mode is `build` and then check if its lib mode.

Reply:
I can try, but Isn't this already importAnalysis**Build**, the version of import analysis that only runs in build mode?